package.json

```diff
@@ -54,7 +54,7 @@
     "prebuild": "./tooling/index.js exec -- rimraf dist",
     "build": "./tooling/index.js build",
     "prebuild:docs": "rimraf ./docs/api",
-    "build:docs": "documentation build --np --project-name ciscospark --project-version 1.x --format html --output ./docs/api --github ./packages/node_modules/ciscospark/src/index.js ./packages/node_modules/@ciscospark/plugin-phone/src/index.js",
+    "build:docs": "documentation build --access public --np --project-name ciscospark --project-version 1.x --format html --output ./docs/api --github ./packages/node_modules/ciscospark/src/index.js ./packages/node_modules/@ciscospark/plugin-phone/src/index.js",
```

Comment:
it prevents anything tagged with `@protected` from being documented. my goal is for `@public` stuff to go into the docs site and `@public + @protected` to go into readmes. That way, the docs site will explain how to use the officially sdk, but the readmes will explain how to use each plugin and fully explain spark-core, etc.